scheduler/src/cook/mesos/api.clj

```diff
@@ -650,79 +677,81 @@
+                 :ports (:job/ports job 0)
+                 :instances
+                 (map (fn [instance]
+                        (let [hostname (:instance/hostname instance)
+                              executor-states (get-executor-states fid hostname)
+                              url-path (try
+                                         (executor-state->url-path hostname (get executor-states (:instance/executor-id instance)))
+                                         (catch Exception e
+                                           nil))
+                              start (:instance/start-time instance)
+                              mesos-start (:instance/mesos-start-time instance)
+                              end (:instance/end-time instance)
+                              cancelled (:instance/cancelled instance)
+                              reason (reason/instance-entity->reason-entity db instance)
+                              base {:task_id (:instance/task-id instance)
+                                    :hostname hostname
+                                    :ports (:instance/ports instance)
+                                    :backfilled false ;; Backfill has been deprecated
+                                    :preempted (:instance/preempted? instance false)
+                                    :slave_id (:instance/slave-id instance)
+                                    :executor_id (:instance/executor-id instance)
+                                    :status (name (:instance/status instance))}
+                              base (if url-path
+                                     (assoc base :output_url url-path)
+                                     base)
+                              base (if start
+                                     (assoc base :start_time (.getTime start))
+                                     base)
+                              base (if mesos-start
+                                     (assoc base :mesos_start_time (.getTime mesos-start))
+                                     base)
+                              base (if end
+                                     (assoc base :end_time (.getTime end))
+                                     base)
+                              base (if cancelled
+                                     (assoc base :cancelled cancelled)
+                                     base)
+                              base (if reason
+                                     (assoc base
+                                       :reason_code (:reason/code reason)
+                                       :reason_string (:reason/string reason))
+                                     base)]
+                          base))
+                      (:job/instance job))}]
+    (cond-> job-map
+            groups (assoc :groups (map #(str (:group/uuid %)) groups))
+            application (assoc :application {:name (:application/name application)
```

Comment:
See [remove-datomic-namespacing](https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/mesos/util.clj#L89) so you don't have to do this manually.